### DISCUSSION TITLE
"Segment"- routable?

### DISCUSSION THREAD

**Discussion by PaudelXA** [OvertureMaps/data#137] [Discussion]
Is the Transportation Data (Segment, Connector) data a routable network? 

(Context: I am using 03-12-2024 parquet file. I believe it is a geo parquet file)

**Comment by jwass** [comment]
Hi @PaudelXA Yes - the network is routable with connectors defining the connections between segments.

**Reply by durkie** [reply]
How does this work with interior connectors (as mentioned here: https://docs.overturemaps.org/guides/transportation/shape-connectivity/#interior-connectors )? I don't understand how the GERS IDs can remain constant with the addition of a new connecting east-west road (as in the example on that webpage) and the network remain routable.
